doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,123 @@ a single confidential command to let the redirection also apply for 'set -x'.
+```
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option
+      (e.g. DASD, SCSI, ...). If you want to use a tape device, see `zipl(8)`
+      to use the `--tape` option instead. Note that the `--tape` option with
+      ReaR is untested so contributions to this guide are welcome! Also, it is
+      not possible to use `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory, in our case
+           `/dev/dasda`!
+
+Use the following command to make the device mounted at `/mnt/bootdisk` bootable
+with `zipl`:
+```
+# zipl --target=/mnt/bootdisk \
+       --parameters='root=/dev/ram0 ro' \
+       --image=/mnt/bootdisk/kernel-HOSTNAME \
+       --ramdisk=/mnt/bootdisk/initramfs-HOSTNAME.img
+
+Run /lib/s390-tools//zipl_helper.device-mapper /mnt/bootdisk/
+Building bootmap in '/mnt/bootdisk/'
+Adding IPL section
+  initial ramdisk...: /mnt/bootdisk/initramfs-HOSTNAME.img
+  kernel image......: /mnt/bootdisk/kernel-HOSTNAME
+  kernel parmline...: 'root=/dev/ram0 ro'
+  component address:
+    internal loader.: 0x0000a000-0x0000dfff
+    parameters......: 0x00009000-0x00009fff
+    kernel image....: 0x00010000-0x0076afff
+    parmline........: 0x0076c000-0x0076cfff
+    initial ramdisk.: 0x00780000-0x28708fff
+    environment blk.: 0x0077c000-0x0077cfff
+Preparing boot device for CCW- and LD-IPL: dasda.
+Done.
+```
+
+If you want to boot the device with ReaR recovery from a running Linux
+instance, you can use the `chreipl` tool to select a new default boot device
+and reboot:
+```
+# chreipl /mnt/bootdisk
+Re-IPL type: ccw
+Device:      0.0.0120
+Loadparm:    ""
+Bootparms:   ""
+clear:       0
+# reboot
+...
+```
+
+If you do not have a running Linux instance at hand, you can force the boot
+from the given device using the z/VM Control Program (CP) directly, e.g. through
+a terminal connection.
+
+The following example only considers DASD disk devices. Consult the official
+z/VM CP Manual for booting from other devices (e.g. FCP attached SCSI) or for
+additional details in general.
+
+Use the following command to boot from a DASD boot device through z/VM CP,
+where the last argument corresponds to the virtual device number (`vdev`).
+```
+CP IPL 0120
+```
+
+The virtual device number can be either obtained from the `Bus-ID` column using
+the `lsdasd -s` command from a running Linux instance:
+```
+# lsdasd -s
+Bus-ID    Status    Name      Device  Type         BlkSz  Size      Blocks
+================================================================================
+0120      active    dasda     94:0    ECKD         4096   39128MB   10017000
+```
+or directly from z/VM CP with the following query:
+```
+CP QUERY VIRTUAL DASD
+00:
+00: CP QUERY VIRTUAL DASD
+00: DASD 0120 ON DASD  2213 R/W 0X0120 SUBCHANNEL = 0004
+00: DASD 0121 ON DASD  201A R/W 0X0121 SUBCHANNEL = 0005
```

Comment:
Not part of the documentation changes, but I think it could be useful to set the label of the dasd formatted by ReaR to e.g. `REAR` so that it is immediately visible in `CP`/`CMS` that it contains the ReaR backup. This would belong to a hypothetical `USB` output method for DASDs, specifically the `rear format` step.